src/main/java/com/example/demo/repositories/AuctionRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.example.demo.repositories;
+
+import com.example.demo.entities.Auction;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AuctionRepository extends PagingAndSortingRepository<Auction,Long>, CrudRepository<Auction,Long> {
```

Comment:
why extending CrudRepo? doesn't PagingAndSorting have everything you need?

Reply:
PagingAndSorting includes only two findAll methods for Page's and Sort's. I will remove PagingAndSorting in repositories which don't use it in future